scripts/build_helpers/bin/build_dart.dart

```diff
@@ -8,14 +8,33 @@ import 'package:path/path.dart' as path;
 
 void main(List<String> args) async {
   final parser = ArgParser();
-  parser.addFlag('verbose', abbr: 'v');
+  parser.addFlag('verbose', abbr: 'v', help: 'enable all debug');
+  parser.addOption('buildType',
+      abbr: 't',
+      help: 'build typ release or debug. all for both',
+      allowed: ['all', 'debug', 'release']);
+
+  ArgResults? argResults;
+  try {
+    argResults = parser.parse(args);
+  } catch (error) {
+    if (error is! FormatException) rethrow;
+    print(parser.usage);
+    exit(-1);
+  }
 
-  final argResults = parser.parse(args);
   Level logLevel = Level.info;
   if (argResults['verbose'] == true) {
-    logLevel = Level.debug;
+    logLevel = Level.all;
   }
-  BuildToolsLogger.initLogger(logLevel: logLevel);
+
+  BuildToolsLogger.initLogger(
+    logLevel: logLevel,
+  );
+
+  String buildType = argResults['buildType'] ?? "all";
```

Comment:
I think the default should be `release` not `all`

Reply:
Default working without ?? all i think. it looks on my tests now